Packs/FeedAzureADConnectHealth/Integrations/AzureADConnectHealthFeed/AzureADConnectHealthFeed.py

```diff
@@ -0,0 +1,202 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+from typing import Any, Callable, Dict, List, Tuple
+
+import urllib3
+from bs4 import BeautifulSoup
+import re
+
+# disable insecure warnings
+urllib3.disable_warnings()
+
+INTEGRATION_NAME = 'Microsoft Azure AD Connect Health Feed'
+
+
+class Client(BaseClient):
+    """
+    Client to use in the Microsoft Azure Feed integration. Overrides BaseClient.
+    """
+
+    def __init__(self, base_url: str, verify: bool = False, proxy: bool = False):
+        """
+        Implements class for Microsoft Azure feeds.
+        :param url: the Azure endpoint URL
+        :verify: boolean, if *false* feed HTTPS server certificate is verified. Default: *false*
+        :param proxy: boolean, if *false* feed HTTPS server certificate will not use proxies. Default: *false*
+        """
+        super().__init__(base_url, verify=verify, proxy=proxy)
+
+    def build_iterator(self) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+        result = []
+        r = self._http_request('GET', url_suffix='', full_url=self._base_url, resp_type='text')
+
+        soup = BeautifulSoup(r, 'html.parser')
+
+        def subs(text):
+            """Cleans up the parsed entries.
+            Returns:
+                A string.
+            """
+            patterns = (('comp', 'com p'), ('comm', 'com m'), ('comf', 'com f'), ('\n', ''))
+            for e in patterns:
+                text = re.sub(e[0], e[1], text)
```

Comment:
why do you need to use regex replacements? your patterns are all constants.
```suggestion
            patterns = (('comp', 'com p'), ('comm', 'com m'), ('comf', 'com f'), ('\n', ''))
            for old, new in patterns:
                text = text.replace(old, new)
```
or you can still use regex
```suggestion
            text = re.sub('com([pmf])', r'com \1', text)
            text = text.replace('\n', '')
```

Reply:
Good suggestions. I like the first one better, made the change